private/bufpkg/bufanalysis/print.go

```diff
@@ -245,3 +255,10 @@ func printEachAnnotationOnNewLine(
 	}
 	return nil
 }
+
+func atLeast1(i int) int {
```

Comment:
nit: you can use this in [fileAnnotation#String](https://github.com/bufbuild/buf/blob/main/private/bufpkg/bufanalysis/file_annotation.go#L91-L96) as well.

Reply:
done